UnityProject/Assets/Scripts/Player/PlayerCrafting.cs

```diff
@@ -102,7 +105,16 @@ public List<CraftingRecipe> GetKnownRecipesInCategory(RecipeCategory recipeCateg
 		/// <returns>True, if a player already knows the recipe, false otherwise.</returns>
 		public bool KnowsRecipe(CraftingRecipe recipe)
 		{
-			return GetKnownRecipesInCategory(recipe.Category).Contains(recipe);
+			try
+			{
+				return GetKnownRecipesInCategory(recipe.Category).Contains(recipe);
+			}
+			catch(Exception e)
+			{
+				Console.WriteLine(e.StackTrace);
```

Comment:
log it instead

Reply:
Sorry, it's removed now. I was using this for debugging